observables/exercise-3/observables.js

```diff
@@ -17,3 +17,8 @@ const clickStream = Observable.fromEvent(theButton, "click");
     No output after the fourth click,
     "Dog #5" after the fith click, and so on.
 */
+
```

Comment:
Note: we may want to filter before we map, so we avoid doing extra work on the emissions we'll eventually ignore. But since we are actually adding the source emission's number to the string, we have to filter **after** we map.

For example, if we filtered first and then mapped, then the output would be `Dog #1, Dog #2`, etc. If we map and then filter, the output is `Dog #1, Dog #3`, etc